src/featureflagservice/lib/featureflagservice/feature_flags.ex

```diff
@@ -35,14 +35,14 @@ defmodule Featureflagservice.FeatureFlags do
 
   ## Examples
 
-      iex> get_feature_flag!(123)
+      iex> get_feature_flag!(foo)
       %FeatureFlag{}
 
-      iex> get_feature_flag!(456)
+      iex> get_feature_flag!(bar)
       ** (Ecto.NoResultsError)
 
   """
-  def get_feature_flag!(id), do: Repo.get!(FeatureFlag, id)
+  def get_feature_flag!(name), do: Repo.get!(FeatureFlag, name)
```

Comment:
I think we already have this method implemented below:
https://github.com/open-telemetry/opentelemetry-demo/pull/1319/files#diff-1ccc92918885eea6add5267ae19facac19aff70e3da8a6d2b5fc7260b962f840R59

Reply:
The Elixir Etco framework requires this call. I tried removing it and got an error.